.travis.yml

```diff
@@ -37,4 +37,3 @@ addons:
     packages:
       - xvfb
       - g++-4.8
-      - libgnome-keyring-dev
```

Comment:
unless you happened to be the 3%
https://bugs.chromium.org/p/chromium/issues/detail?id=466975